workflows/utilities/wf_flu_track.wdl

```diff
@@ -112,6 +112,14 @@ workflow flu_track {
         docker = assembly_metrics_docker
     }
   }
+  # Calculate the percentage of mapped reads for flu samples
+  if (defined(irma.seg_ha_bam) || defined(irma.seg_na_bam)) {
```

Comment:
I think we would want to know the percentage of mapped reads for both of these segments. 

We are also already running this task individually for both the HA and NA segment, so could you please do the following:

- [ ] remove this block (lines 116-122)
- [ ] add a line under the `String ha_na_assembly_coverage_string` line that performs the same functionality but for the percentage_mapped_reads output
- [ ] Convert the percentage_mapped_reads outputs in both this task and the other workflows that have flu to a String in order to accommodate this change
  - [ ] wf_flu_track.wdl
  - [ ] wf_theiacov_ont.wdl
  - [ ] wf_theiacov_illumina_pe.wdl

After that, I think this PR will be completely ready for merge!

Reply:
That should be updated now, theiacov ont and illumina pe were already set to String